L/likwid/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "likwid"
+version = v"5.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/RRZE-HPC/likwid/archive/refs/tags/v$(version).tar.gz", "1b8e668da117f24302a344596336eca2c69d2bc2f49fa228ca41ea0688f6cbc2"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd likwid-5.2.1/
```

Comment:
This is more robust against updates (avoid hardcoding version numbers everywhere)
```suggestion
cd $WORKSPACE/srcdir/likwid*
atomic_patch -p2 ../patches/perfevents_hasschedaffinity_exthwlocfix.patch
```

Reply:
Just to be safe, you did change `-p1` to `-p2`. Was that intentional? (I have no idea what this does 😄  This patch stuff has been autogenerated by the wizard, as you probably know.)

Reply:
That's the number of directory levels to strip from the patch.  The patch references files like `before/likwid-5.2.1/Makefile`, I _**do**_ want to strip also `likwid-5.1.2` because (1) now the patch is applied from within the likwid source directory (2) I don't want to deal with hardcoded version number in the patch